test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -261,7 +261,7 @@ Date: REGEX(.*)
 04. Statistics
 ==============
 HTTP/1.1 200 OK
-Content-Length: 22REGEX(\d)
+Content-Length: REGEX(\d+)
```

Comment:
Why this PR makes that change needed?

Reply:
I don't think the problem is the PR, except for changes in timing. The broker gets a little faster with this PR.
The problem seems like something completely different, something we need to look into

Reply:
Same same,

NTC (for now)

Reply:
This is also related with https://github.com/telefonicaid/fiware-orion/issues/3258 ? In that case, the body of the issue should mention this .test file, in order to fix it.

Reply:
Added to issue: https://github.com/telefonicaid/fiware-orion/issues/3258#issuecomment-410240971